kpi/deployment_backends/mock_backend.py

```diff
@@ -201,6 +205,53 @@ def duplicate_submission(
         settings.MONGO_DB.instances.insert_one(duplicated_submission)
         return duplicated_submission
 
+    def get_attachment_content(self, user, submission_uuid, response_xpath):
+
+        try:
+            submission_xml = self.get_submissions(
+                user, format_type=SUBMISSION_FORMAT_TYPE_XML, query={
+                    '_uuid': submission_uuid
+                }
+            )
+            first_submission = submission_xml[0]
+        except StopIteration:
```

Comment:
🤔 , I don't think it's going to return a `StopIteration` error. 
Maybe IndexError  

```python
        submission_xml = self.get_submissions(
            user, format_type=SUBMISSION_FORMAT_TYPE_XML, query={
                '_uuid': submission_uuid
            }
        )
        try:
            first_submission = submission_xml[0]
        except IndexError:
```